### PR TITLE
Reduce immediate string values.

### PR DESCRIPTION
**Pull request by cryptcoin-junkey**
This patch privides two benefits.
1. Reduces bugs by typo.
2. Helps porting Counterparty to altcoins like Dogecoin, Monacoin.

### CODE REVIEW

File: src/js/components/balances.js [hunk 1]
```diff
@@ -1377,15 +1377,15 @@ function SweepModalViewModel() {
             PENDING_ACTION_FEED.add(sendTxHash, "sends", sendData);
 
             // here we adjust the BTC balance whith the change output
-            if (selectedAsset.ASSET != 'BTC') {
+            if (selectedAsset.ASSET !== KEY_ASSET.BTC) {
               var newBtcBalance = CWBitcore.extractChangeTxoutValue(sendData.source, unsignedTxHex);
-              $.jqlog.debug("New BTC balance: " + newBtcBalance);
+              $.jqlog.debug("New ' + KEY_ASSET.BTC + ' balance: " + newBtcBalance);
```
Comment (deweller) [inline src/js/components/balances.js]: need double quotes here

File: src/js/components/history.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ function BalanceHistoryViewModel() {
         }).error(function(jqXHR, textStatus, errorThrown) {
           var address = /address%3D([A-Za-z0-9]+)%22/g.exec(jqXHR.url)[1];
           var addressHash = hashToB64(address);
-          $.jqlog.debug("Could not get BTC balance from blockchain for address " + address + ": " + errorThrown);
+          $.jqlog.debug("Could not get ' + KEY_ASSET.BTC + ' balance from blockchain for address " + address + ": " + errorThrown);
```
Comment (deweller) [inline src/js/components/history.js]: Same here.  These should be double quotes.

File: src/js/components/wallet.js [hunk 3]
```diff
@@ -392,10 +392,10 @@ function WalletViewModel() {
         // the (confirmed) balance will be decreased by the ENTIRE quantity of that txout, even though they may be getting
         // some/most of it back as change. To avoid people being confused over this, with BTC in particular, we should
         // display the unconfirmed portion of the balance in addition to the confirmed balance, as it will include the change output
-        self.updateBalance(data[i]['addr'], "BTC", data[i]['confirmedRawBal'], data[i]['unconfirmedRawBal']);
+        self.updateBalance(data[i]['addr'], KEY_ASSET.BTC, data[i]['confirmedRawBal'], data[i]['unconfirmedRawBal']);
 
         addressObj = self.getAddressObj(data[i]['addr']);
-        assert(addressObj, "Cannot find address in wallet for refreshing BTC balances!");
+        assert(addressObj, "Cannot find address in wallet for refreshing ' + KEY_ASSET.BTC + ' balances!");
```
Comment (deweller) [inline src/js/components/wallet.js]: Need double quotes here.

File: src/js/consts.js [hunk 4]
```diff
@@ -251,3 +251,9 @@ var TRANSACTION_MAX_RETRY = 5; // max retry when transaction failed (don't inclu
 var DONATION_ADDRESS = USE_TESTNET ? 'n4MGGJBkW9RjRKBbZfBAceHDndhywvVPV9' : '19U6MmLLumsqxXSBMB5FgYXbezgXYC6Gpe';
 
 var APPROX_SECONDS_PER_BLOCK = USE_TESTNET ? 20 * 60 : 8 * 60; //a *rough* estimate on how many seconds per each block (used for estimating open order time left until expiration, etc)
+
+var KEY_ASSET = {
+  'BTC': 'BTC',
+  'XCP': "XCP'
```
Comment (deweller) [inline src/js/consts.js]: This double quote should be a single quote

File: src/js/components/balances.js (hunk 1 above)
Comment (cryptcoin-junkey) [reply src/js/components/balances.js]: Thank you for your review.  I'll fix them and re-push within a few days.

**Comment by cryptcoin-junkey** [comment]
I fixed some typos @deweller pointed. And I found some more fixes for the better altcoin support.
I'm fixing now. Please keep this hold.

**Comment by cryptcoin-junkey** [comment]
@deweller I finished fixes. Tested by my hand as I couldn't find e2e test cases.

And I'm developing Monaparty that is a Counterparty port using Monacoin blockchain.
I run Monaparty version of counterwallet with this PR and works well.

**Pull request merged**